src/sentry/static/sentry/app/components/debugFileFeature.tsx

```diff
@@ -63,6 +63,10 @@ DebugFileFeature.propTypes = {
   feature: PropTypes.oneOf(Object.keys(FEATURE_TOOLTIPS)).isRequired,
 };
 
+DebugFileFeature.defaultProps = {
+  available: true,
```

Comment:
As this is a function component, I would provide default prop via ES6 default parameters.